src/compas/datastructures/datastructure.py

```diff
@@ -21,6 +21,41 @@ def __init__(self, attributes=None, name=None):
         self._aabb = None
         self._obb = None
 
+    def __get_mro__(self):
```

Comment:
`__clstree__` or `__inheritance__` or something like that?

Reply:
ok, `__inheritance__`  and as property